guides/source/action_cable_overview.md

```diff
@@ -659,6 +659,9 @@ Below is a list of the subscription adapters available for end users.
 
 The async adapter is intended for development/testing and should not be used in production.
 
+It works within a single process only, so even in development, you won't be able to broadcast
+from the Rails console and receive it in the Rails web server. For that, use one of the other adapters.
```

Comment:
While I use the redis adapter in dev, I do think this means we should fix the Async adapter instead. @gmcgibbon something you're interested in with your other adapter related work?

Reply:
👍 I'll take a look.

Reply:
@gmcgibbon Hi! Did anything come of this? Just asking because this doc PR was auto-closed for inactivity.